src/components/_/TipTap.vue

```diff
@@ -236,6 +269,9 @@ de:
   redo: Wiederholen
   strike: Durchgestrichen
   undo: Rückgängig machen
+  formatAlignLeft: Linksbündig
+  formatAlignRight: Rechtsbündig
+  formatAlignCenter: Mittig
```

Comment:
Would you keep the nice alphabetical sorting of translation keys? :pleading_face:

Reply:
done